test/integration/managed_instance_group/inspec.yml

```diff
@@ -0,0 +1,64 @@
+name: default
+attributes:
```

Comment:
Were you able to use a new enough version of InSpec to use these global attributes, or are these here for when kitchen-terraform relaxes their InSpec version pin?

Reply:
@adrienthebo Push coming with the latest and greatest InSpec :)

Reply:
💯